src/components/RevokeKeyDialog/RevokeKeyDialog.tsx

```diff
@@ -34,15 +34,19 @@ export const RevokeKeyDialog = ({ row }: RevokeKeyDialogProps) => {
       // At this point, spvWalletClient is defined; using non-null assertion.
       return await spvWalletClient!.RevokeAccessKey(row.original.id);
     },
-    onSuccess: () =>
+    onSuccess: () => {
```

Comment:
For now, there is no such thing as "revoke key" for admin in `spv-wallet`.
So please just remove this button/action from the app (in admin mode).

![image](https://github.com/user-attachments/assets/8d42dec1-f512-4023-b958-2b5a486ed558)

Reply:
It has removed in `_admin.access-keys.tsx` file